manifests/n/NewTek/NDI5Tools/5.1.1.1/NewTek.NDI5Tools.locale.en-US.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.5.0.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: NewTek.NDI5Tools
+PackageVersion: 5.1.1.1
+PackageLocale: en-US
+Publisher: NewTek

```

```suggestion
Publisher: NewTek, inc.
```
This must match Control Panel.